src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/VariantDetectorFromLongReadAlignmentsForSimpleStrandSwitchUnitTest.java

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.hellbender.tools.spark.sv.discovery.prototype;
+
+import htsjdk.samtools.Cigar;
+import htsjdk.samtools.TextCigarCodec;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.AlignmentInterval;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import scala.Tuple2;
+
+import java.io.IOException;
+
+public class VariantDetectorFromLongReadAlignmentsForSimpleStrandSwitchUnitTest extends BaseTest {
```

Comment:
It might be worth adding a test for `extractCigarElements` as well. Perhaps some of the other methods in the class, too.